loleaflet/css/mobilewizard.css

```diff
@@ -665,10 +665,6 @@ a.leaflet-control-zoom-in {
 #buttonnone + label, #buttonbefore + label, #buttonafter + label, #buttonoptimal + label, #buttonparallel + label, #buttonthrough + label{
 	padding-left: 0;
 }
-#Color > img{
```

Comment:
please change this line to:
"#mobile-wizard #Color > img{"
this should limit it to the mobile-wizard only